SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -259,7 +259,7 @@ bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 
 	if (IsManualWorkerConnectionSet(GetPrimaryLaunchConfigPath()))
 	{
-		if (!FMessageDialog::Open(EAppMsgType::YesNo, LOCTEXT("AllowManualWorkerConnection", "Chosen launch configuration will not automatically launch servers. Do you want to continue?")) == EAppReturnType::Yes)
+		if ((!FMessageDialog::Open(EAppMsgType::YesNo, LOCTEXT("AllowManualWorkerConnection", "Chosen launch configuration will not automatically launch servers. Do you want to continue?"))) == EAppReturnType::Yes)
```

Comment:
How come this extra bracket?

Reply:
it would fail to build in XCode because it's ambigious on whether the ! should be only applied to the left-hand side or the overall expression

Reply:
though we probably should just change the `==` to a`!=`

Reply:
Yeah let's do !=